src/Signature.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PharIo\GnuPG;
+
+/**
+ * This class could be prevent sigatures form being serialized.
+ *
+ * @see https://github.com/paragonie/hidden-string/blob/master/src/HiddenString.php inspired by.
+ */
+class Signature
```

Comment:
To me, a `Signature` is more than just the raw string.

We should probably rename this class to something that represents this implementation better and create an actual `Signature` class that can be the return type for `sign` and contains as one property whatever this class gets renamed to to represent the signature data.